### ISSUE TITLE
History of iodine ise

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9815]
**Preferred term label:**
History of iodine consumption

**Synonyms**
Use of iodinated contrast medium; radioiodine therapy; ingestion

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Thyroid storm

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000293
- label: History of recent iodated contrast medium administration
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: PMID:23148056|PMID:33948409
- def: Medical history of recent iodinated or radioiodine contrast use.
- comment: Iodinated contrast media commonly used in various radiologic examinations and interventional procedures results in massive iodide exposure to the thyroid gland. Whereas people with normal thyroids usually experience no ill effects, those with pre-existing thyroid disease may experience thyrotoxicosis (often referred to as thyroid storm in this context), which, if unrecognized, could lead to serious, potentially life-threatening consequences, including atrial arrhythmias, heart failure, pulmonary arterial hypertension, cerebrovascular and pulmonary embolism, and cardiomyopathy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9815
- synonyms:
